demo/SalesDemoApp/SalesDemoApp.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,6 @@
 	objects = {
```

Comment:
Do we need to push this file to repo ?

Reply:
I have updated the project settings for the sales test app as well, so we don't need to do these changes everytime for testing.